plugins/scaffolder-backend/src/scaffolder/stages/publish/github.test.ts

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+jest.mock('@octokit/rest');
+jest.mock('nodegit');
```

Comment:
just out of curiosity, you put them here for clarity, not because of lacking hoisting in the build step right? i kind of like it

Reply:
yep, it's just nice to document at the top what it's going to do before imports.